src/neuroconv/tools/nwb_helpers/_metadata_and_file_helpers.py

```diff
@@ -221,27 +222,35 @@ def make_or_load_nwbfile(
 
         io = backend_io_class(**load_kwargs)
 
+    nwbfile_loaded_succesfully = True
     try:
         if load_kwargs.get("mode", "") == "r+":
             nwbfile = io.read()
         elif nwbfile is None:
             nwbfile = make_nwbfile_from_metadata(metadata=metadata)
         yield nwbfile
-    except Exception as e:
-        success = False
-        raise e
+    except Exception as load_error:
+        nwbfile_loaded_succesfully = False
```

Comment:
Could have been a problem with `metadata` in `make_nwbfile_from_metadata` that is getting caught in this block, separate from file read

Reply:
Yep. I think that could be that's why I am calling ti "load_error" instead of "create_error" to cover both read and create_with_metadata errors.